### PR TITLE
Add Windows code to kill a stuck server.

### PR DESCRIPTION
**Pull request by bjones1**

### CODE REVIEW

File: runestone/unittest_base.py [hunk 1]
```diff
@@ -36,9 +37,14 @@
...
 PORT = "8081"
 # Use the localhost for testing.
-HOST = "http://127.0.0.1:" + PORT
+HOST_ADDRESS = "127.0.0.1:" + PORT
+HOST_URL = "http://" + HOST_ADDRESS
 
 
+# Define the platform.
+IS_WINDOWS = platform.system() == "Windows"
+IS_LINUX = sys.platform.startswith("linux")
```
Comment (bnmnetp) [inline runestone/unittest_base.py]: could you make this linux or darwin ? this should cover the macOS folks with out any problems.

Comment (bjones1) [reply runestone/unittest_base.py]: Are you sure? I use this only to determine if we should run pyvirtualdisplay, and I can't tell if mac supports that or not. See line 149.

Comment (bnmnetp) [reply runestone/unittest_base.py]: I was just trying to make sure that we didn't include windows by excluding macOS.  the pyvirtualdisplay issue probably doesn't matter for Mac.

Comment (bjones1) [reply runestone/unittest_base.py]: Since the only use of `IS_LINUX` is for pyvirualdisplay, I'd say leave this as it is. Sound good?

**Pull request merged**